apps/total-typescript/src/components/team/self-redeem-button.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {handleSelfRedeem} from 'path-to-purchase-react/handle-self-redeem'
+import {Button} from '@skillrecordings/react'
+import {Purchase} from '@skillrecordings/database'
+
+const SelfRedeemButton: React.FC<
```

Comment:
*not this PR*: is there a shared place for a simple, reused component like this to live and be shared across apps? Perhaps `packages/react/src/components`? Not sure what precedent we have for this, but this component seems like a good one to experiment with.

Reply:
my intent here was to suggest a specific package for this stuff and not cram it into an existing catch all package

having many packages is good imo